### ISSUE TITLE
hack for ios

### ISSUE THREAD

**Issue by cityrhath1** [thatskymod/Sky-CotL-Scripts#23]
can you made hack for ios as deb file

**Comment by thatskymod** [comment]
No, as a lifelong Android user, I can't see myself making iOS programs anytime soon.

Here are the problems with creating mods for iOS:

- Market share. Apple users make up a minority of modders, so modding on iOS may not be as profitable as on Android.
- Security limitations. Apple iOS has restrictions in terms of what you can access and customize on a software level.

Due to the restrictions on iOS, including its closed system, it can be harder for developers to create mods that work on Apple's mobile platform. This can limit the availability and functionality of mods for iOS users.

In conclusion, I personally will not be creating mods for iOS. However, if you know of any usable iOS mods, I would be happy to learn about them!

**Comment by thatskymod** [comment]
No, as a lifelong Android user, I can't see myself making iOS programs anytime soon.

Here are the problems with creating mods for iOS:

- Market share. Apple users make up a minority of modders, so modding on iOS may not be as profitable as on Android.
- Security limitations. Apple iOS has restrictions in terms of what you can access and customize on a software level.

Due to the restrictions on iOS, including its closed system, it can be harder for developers to create mods that work on Apple's mobile platform. This can limit the availability and functionality of mods for iOS users.

In conclusion, I personally will not be creating mods for iOS. However, if you know of any usable iOS mods, I would be happy to learn about them!

**Comment by cityrhath1** [comment]
like this one for standoff 2 
h5gg.searchNumber('81323', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('71003', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('170016', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('3005', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('126300', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('65101', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('44401', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('54401', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('15007', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('[1131500](tel:1131500)', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('81214', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('81200', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('121100', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('11002', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('125200', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('[1075205](tel:1075205)', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('84906', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('150', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('43401', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('47504', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('124500', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('72002', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('81108', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('170023', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('45302', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('97300', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('44902', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('3001', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('43502', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('3007', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('170004', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('157100', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('170012', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('157500', 'I64');
h5gg.clearResults();
h5gg.clearResults();
h5gg.searchNumber('86317', 'I64', '0x00000000', '0x200000000');
h5gg.getResults(100);
h5gg.editAll('47503', 'I64');
h5gg.clearResults();
h5gg.clearResults();



it’s js script if you can make for sky it’s will work on ios

**Comment by thatskymod** [comment]
I see, thanks for sharing!

Actually, that syntax looks very similar to the one that GameGuardian uses, so it might indeed be possible.

Although, the slightest change in compiler options and processor architecture might greatly affect how game data is accessed and manipulated. So, there might be more involved than just converting Lua to JavaScript.

**Comment by thatskymod** [comment]
If you are looking for a modded version of Sky for iOS, one possible resource is https:\/\/iosgods.com/topic/171061-sky-children-of-the-light-cheats-v0211-84/#comment-5404752

However, please note that I have no means of promoting paid content or content hidden behind a paywall, including soft subscription walls that require registration. Use at your own discretion.
